### DISCUSSION TITLE
Campaign Needs To Be Able To Save More

### DISCUSSION THREAD

**Discussion by VaelophisNyx** [FakeFishGames/Barotrauma#12444] [Ideas and suggestions]
Currently it can be an exhausting task to get to an outpost to save the game. Late biomes have so few outposts that it can take hours to get between them. It'd be cool if we could save at empty zones at a minimum...

**Comment by Hazmat616** [comment]
Woah I didn't know we couldn't. That's scary. Perhaps if it's an issue or soft saves at empty locations so in the menu next to retry you can go back to the last outpost.

**Comment by Vykn** [comment]
The game saves every time a level transition occurs. You don't need to go to an outpost to save.

> It'd be cool if we could save at empty zones at a minimum...

That is literally how it works

**Reply by VaelophisNyx** [reply]
It's entirely unstated so that'd be why I opened this. Completely unaware
